sdgym/summary.py

```diff
@@ -72,11 +72,18 @@ def _beat_baseline(data, baseline_scores):
         _synthesizer_beat_baseline, baseline_scores=baseline_scores)
 
 
+def _best_time(data, rank):
```

Comment:
I think this could be merged with the `_best` function above by accepting the field name as input, which would make reading easier because developers only need to learn 1 approach.

The function becomes something like this:

```python
def _best(data, rank, field):
    ranks = data.groupby('dataset').rank(method='dense', ascending=False)[field] == rank
    return ranks.groupby(data.synthesizer).sum()
```

In it can be used as:

```python3
        'best': _best(no_identity, 1, 'normalized_score'),
        'best_time': _best_time(no_identity, 1, 'model_time'),
        'second_best_time': _best_time(no_identity, 2, 'model_time'),
        'third_best_time': _best_time(no_identity, 3, 'model_time'),
```